ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp.j2

```diff
@@ -7,4 +7,10 @@ router_bgp:
 
 {%     include 'network_services/router-bgp-vlan-aware-bundles.j2' %}
 
+{%     if default_vrf.svi_subnets %}
+
```

Comment:
Nitpicking here but I think we only need extra blank lines just after an include statement. So not needed on 11 and 15.